resourcemgr/resourcemgr.c

```diff
@@ -1309,7 +1309,8 @@ TSS2_RC ResourceMgrSendTpmCommand(
                 goto SendCommand;
             }
 
-            if( foundEntryPtr->connectionId != cmdConnectionId )
+            if( foundEntryPtr->status.persistent == 0 &&
+                foundEntryPtr->connectionId != cmdConnectionId )
```

Comment:
I do not agree with this change.  There is nothing in the RM spec that allows sharing of persistent objects between connections.
What's the problem you're trying to solve here?